app/controllers/invitations_global_controller.rb

```diff
@@ -0,0 +1,45 @@
+class InvitationsGlobalController < ApplicationController
```

Comment:
@dissident I think it will be better to name it just `InvitationsController`. Same for model. I know that we already have model with `Invitation` name it may produce some issues. But it will be better to name it right now.

BTW, can we combine both models? Just use empty `organization_id` field for global invitations.